tasks/build.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+- name: Create Prometheus Node Exporter directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ prometheus_node_exporter.user }}"
+    group: "{{ prometheus_node_exporter.group }}"
+    mode: 0750
+  with_items:
+    - "{{ prometheus_node_exporter.path.install }}"
+    - "{{ prometheus_node_exporter.path.log }}"
+
+- name: Copy wrapper shell script
+  become: yes
+  copy:
+    src: prometheus_exporter.sh
+    dest: "{{ prometheus_node_exporter.path.install }}/prometheus_exporter.sh"
+    owner: "{{ prometheus_node_exporter.user }}"
+    group: "{{ prometheus_node_exporter.group }}"
+    mode: 0755
+
+- stat:
+    path: "{{ prometheus_node_exporter.download_dest }}/{{ prometheus_node_exporter.download_url[70:] }}"
```

Comment:
Can you try splitting the string rather than using a hard code number? Eg.

```YAML
path: "{{ prometheus_node_exporter.download_dest }}/{{ prometheus_node_exporter.download_url.split('/') | last }}"
```